### DISCUSSION TITLE
Tilemap in Bevy 0.10

### DISCUSSION THREAD

**Discussion by Sierra100010** [bevyengine/bevy#8055] [Q&A]
Hi all, I have a question that I can't find a complete answer to in the new version of Bevy. How can I create a tilemap in Bevy? As far as I understand, I have to use MaterialMesh2dBundle to create the mesh for the tilemap? How do I then apply textures to the tilemaps?

**Comment by hymm** [comment]
I would normally recommend bevy_ecs_tilemap, but it hasn't updated to 0.10 yet. There is an open PR for it though https://github.com/StarArawn/bevy_ecs_tilemap/pull/390.
